dashboard/src/stores/package.ts

```diff
@@ -44,6 +47,10 @@ export const usePackageStore = defineStore("package", {
     ui: {
       download: new UIRequest(),
     },
+
+    filters: {
+      status: undefined as PackageListStatusEnum | undefined,
```

Comment:
With the changes mentioned above, if you modify this line to read `status: "" as PackageListStatusEnum`, the select will be populated with `any` on page load.